doc/Extensions/Billing-Module.md

```diff
@@ -36,3 +36,21 @@ $config['billing_data_purge'] = 12;     // Number of months to retain
 Data for the last complete billing cycle will always be retained - only data older than
 this by the configured number of months will be removed.  This task is performed in the
 daily cleanup tasks.
+
+## 95th Percentile Calculation
+
+For 95th Percentile billing, the default behavior is to use the highest of the input 
+or output 95th Percentile calculation.
+
+To instead use the combined total of inout + output to derive the 95th percentile,
+This can be changed on a per bill basis by setting 95th Calculation to "Aggregate".
+
+To change the default option to Aggregate,
+add the following the `config.php`:
+
+```php
+$config['billing']['95th_default_agg'] = 1;  // Set aggregate 95th as default
```

Comment:
I think it would be better to allow all three types to be set as the default in the config.
`$config['billing']['95th_default'] = 'agg';`

Reply:
There's only really 2 types.  Max of in/out, or aggregate total.  

The existing codebase sets direction to in or out based on the highest value, so I'm setting that field to 'agg' (which obviously doesn't change) to override that behavior.  

For the standard functionality, a new bill gets set to 'in' by default and then the billing-calculate script changes it based on real values once there's sufficient data to set a max direction for that billing cycle.

That said, the config knob you suggest would probably make the config more readable, so its not a bad idea.  That'd be a pretty simple change

Reply:
Sounds good, yeah, somehow there only be two slipped my mind when I wrote this. The in/out makes sense as you don't know which direction is "Down" or "Up".